onnxruntime/hosting/server_configuration.h

```diff
@@ -84,9 +91,14 @@ class ServerConfiguration {
         << desc << std::endl;
   }
 
+  inline bool file_exists(const std::string& fileName) {
+    std::ifstream infile(fileName.c_str());
+    return infile.good();
+  }
+
   po::options_description desc{"Allowed options"};
   po::variables_map vm{};
-};
+};  // namespace hosting
```

Comment:
Remove the `// namespace hosting`

Reply:
Ah clang-format added up that when I ran it with too many braces. Removed.